app/presenters/claim_csv_presenter.rb

```diff
@@ -61,15 +72,27 @@ def case_type_name
   end
 
   def scheme
-    if %w[Claim::AdvocateClaim Claim::AdvocateInterimClaim Claim::AdvocateSupplementaryClaim].include? type
+    if AGFS_CLAIM_TYPES.include? type
```

Comment:
so we don't have to repeat the claim types wold be good to have something like

```suggestion
    if  claim.class.agfs_claim_types.map(&:to_s).include? type
```

could tidy further with some kind of delegation - rails' delegate or forwardable
or manual as below
```
private
def agfs_claim_types
  claim.class.agfs_claim_types.map(&:to_s)
end
```

Reply:
i think you can actually do 
```
if claim.class.agfs?...
```